public/app/processing-overlay.js

```
@@ -291,6 +304,67 @@
+      var py = cy + radial * Math.sin(pa.angle);
+      pts.push({ x: px, y: py });
+      ctx.beginPath();
+      ctx.arc(px, py, pa.size, 0, Math.PI * 2);
+      ctx.fillStyle   = 'hsla(' + ((hue + pp * 18) % 360) + ',95%,72%,0.92)';
+      ctx.shadowColor = 'hsla(' + ((hue + pp * 18) % 360) + ',95%,72%,0.95)';
+      ctx.shadowBlur  = 10;
+      ctx.fill();
+      ctx.shadowBlur  = 0;
+    }
+    /* Connect nearest neighbours */
+    for (var a = 0; a < pts.length; a++) {
```

ESLint

[error] 'a' is already defined.

(no-redeclare)